Classes/Core/Jwt/SessionHelper.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TYPO3\TestingFramework\Core\Jwt;
+
+use TYPO3\CMS\Core\Information\Typo3Version;
+use TYPO3\CMS\Core\Session\UserSession;
+
+/**
+ * @internal This class is only for cross core compat. This will not be available in next major version.
+ */
+class SessionHelper
+{
+    public function createSessionIdentifier(string $userSessionId): string
```

Comment:
Naming... the return value is not the session identifier (as persisted in DB), it is also just used as cookie value.
So, e.g. `createSessionCookieValue()`

Reply:
done